src_docs/manual/en/HowTo_UseTeamProject.xml

```diff
@@ -208,10 +214,10 @@
 
       <listitem>
         <warning>
-		  <para>Only the project manager should use <link
+		  <para>Only the project manager should use the <link
 		  endterm="menus.project.title" linkend="menus.project"/><link
 		  endterm="menus.project.commit.source.files.title"
-		  linkend="menus.project.commit.source.files"/>.</para>
+		  linkend="menus.project.commit.source.files"/> command.</para>
```

Comment:
Maybe this should be prefaced with "In most workflows" or something? What's the reasoning for this warning?

Reply:
OK, I think I've got something I'm happy with.